### DISCUSSION TITLE
Add "installed" tab in the list packages view under a source view in zebra. To see all installed packages from selected source.

### DISCUSSION THREAD

**Discussion by sukumargv** [wstyres/Zebra#1463] [Ideas]
**Is your feature request related to a problem? Please describe.**

It is always frustrating to identify which packages were installed from a source "XYZ." If I remove this XYZ source, it will leave the packages installed from this source behind. There is no way to locate these packages (if the source was malicious) and remove them.

**Describe the solution you'd like**
Adding an installed tab or a category in XYZ's package listing, it would make it easier to locate these installed packages and uninstall or modify them.

**Describe alternatives you've considered**
I have posted on Reddit for a third party tweak (like Zenumbra), but the community suggested I make a feature request to zebra. 

**Additional context**

![IMG_3058](https://user-images.githubusercontent.com/2417723/87857160-8eb3a480-c8f2-11ea-8ccd-bbafc2672c80.JPG)

**Comment by jorzuniga** [comment]
This it's a necessary option
